k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -1508,6 +1508,10 @@ spec:
                 properties:
                   enabled:
                     type: boolean
+                  logFormat:
+                    type: string
+                  version:
+                    type: string
```

Comment:
@olemarkus Should we go with `version` or `image` in this case? Any thoughts?

Reply:
The consensus is that we would like `image` instead of `version`. Could you update the PR @anthonyhaussman?

Reply:
@anthonyhaussman Please also add `logLevel`.

Reply:
I have added `logLevel` :slightly_smiling_face: